change/@azure-msal-react-e44ef43a-73f0-4eb4-99e6-8cceeb26058f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

No changes to published package 
```suggestion
  "type": "none",
```